src/elfpy/agent.py

```diff
@@ -3,10 +3,10 @@
 """
 
 import logging
-
```

Comment:
newline between stdlib & external imports is a good idea (black should enforce this?)

Reply:
looks like it doesn't, but i can add it